Web/Case Study Assignment-3/CarDekho/login.html

```diff
@@ -0,0 +1,141 @@
+<script
+	src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+<script src="jquery-1.9.1.js"></script>
+<script type="text/javascript" src="scripts/scriptMain.js"></script>
+</head>
+
+<body>
+
+	<header>
+		<br /> <span id="logo">Car Dekho</span>
+	</header>
+
+	<div id="toppanel" style="z-index: 5">
+		<div id="panel">
+			<div class="content clearfix">
+				<div class="left" style="width: 100%">
+					<h2>Log in</h2>
+					<form action="home.html" onsubmit="return validate();">
+						<input id="email" name="email" type="text" placeholder="Username" /><br />
+						<input id="pwd" name="pwd" type="password" placeholder="Password" /><br />
+						<input type="submit" value="Log in" />
+					</form>
+				</div>
+			</div>
+		</div>
+	
+		<div class="tab">
+			<ul class="login">
+				<li class="left">&nbsp;</li>
+				<li id="toggle"><span href="#" class="a" id="toggle-login">Log
+						in</span></li>
+				<li class="right">&nbsp;</li>
+			</ul>
+		</div>
+
+	</div>
+	<script>
+		$('#toggle').click(function() {
+
+			$('#panel').slideToggle();
+
+		});
+	</script>
+
+	<nav>
+		<a href="login.html">Login</a> <span class="space"><a
+			href="about.html">About Us</a></span> <span class="space"><a
+			href="contact.html">Contact Us</a></span>
+	</nav>
+
+
+	<div id="myCarousel" class="carousel slide" data-ride="carousel">
+
+		<ol class="carousel-indicators">
+			<li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+			<li data-target="#myCarousel" data-slide-to="1"></li>
+			<li data-target="#myCarousel" data-slide-to="2"></li>
+			<li data-target="#myCarousel" data-slide-to="3"></li>
+		</ol>
+
+		<!-- Wrapper for slides -->
+		<div class="carousel-inner" role="listbox">
+			<div class="item active">
+				<img src="images/car1.jpg">
+				<div class="carousel-caption">
+					<h3>
+						<font color="white">LUXURY CARS</font>
+					</h3>
+				</div>
+			</div>
+			<div class="item">
+				<img src="images/car2.jpg">
+				<div class="carousel-caption">
+					<h3>
+						<font color="white">NEW AND SECOND HAND</font>
+					</h3>
+				</div>
+			</div>
+			<div class="item">
+				<img src="images/car3.jpg">
+				<div class="carousel-caption">
+					<h3>
+						<font color="white">QUALITY OF SERVICE</font>
+					</h3>
+				</div>
+			</div>
+			<div class="item">
+				<img src="images/car4.jpg">
+				<div class="carousel-caption">
+					<h3>
+						<font color="white">SUPER SPORTS</font>
+					</h3>
+				</div>
+			</div>
+		</div>
+		
+		<a class="left carousel-control" href="#myCarousel" role="button"
+			data-slide="prev"> <span class="glyphicon glyphicon-chevron-left"
+			aria-hidden="true"></span> <span class="sr-only">Previous</span>
+		</a> <a class="right carousel-control" href="#myCarousel" role="button"
+			data-slide="next"> <span
+			class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+			<span class="sr-only">Next</span>
+		</a>
+	</div>
+
+
+
+	<section>
```

Comment:
ACTION: This search section should be placed in header section of the page not in bottom. Check your design